public/dist/inverted-index.js

```diff
@@ -11,10 +11,12 @@ var _createClass = function () { function defineProperties(target, props) { for
 
```

Comment:
This file is here because I am using the trans piled version in Karma configuration file.